environment.yml

```diff
@@ -116,6 +116,15 @@ dependencies:
   - requests
   - pygments # Code highlighting
 
+  # web interactive REPL
+  # Pyodide kernel version 0.5.2 comes with Pyodide 0.27.1
+  # which in-turn comes with pandas 2.2.3:
+  # 1. https://jupyterlite-pyodide-kernel.readthedocs.io/en/stable/#compatibility
+  # 2. https://pyodide.org/en/stable/usage/packages-in-pyodide.html
+  # We constrain/pin here to avoid breaking changes on rebuilds.
```

Comment:
We typically do not do this because hard pinning for some time and then doing a major upgrade is often harder than addressing each break as it comes in. Is there a reason we should take a different approach here?

Reply:
Unpinned in 391ce1ea50b279b53dfc18504f5599c25cf9f568 for now, thanks!